spacy/lang/ru/lemmatizer.py

```diff
@@ -1,7 +1,14 @@
 # coding: utf8
+from __future__ import unicode_literals
+
 from ...symbols import ADJ, DET, NOUN, NUM, PRON, PROPN, PUNCT, VERB, POS
 from ...lemmatizer import Lemmatizer
 
+try:
+    Str = unicode
+except NameError:
+    Str = str
+
```

Comment:
To make this more elegant, you can also import our `unicode_` helper from `spacy.compat` – [see here](https://github.com/explosion/spaCy/blob/develop/spacy/compat.py#L51). Because that type of hacky compatibility check comes up a lot, we've added some helpers for it 🙂

For example:

```python
from ...compat import unicode_

if isinstance(univ_pos, unicode_):
   # and so on
```

Reply:
Thanks for the feedback. At first I wanted to use `six` for that, but since `six` not specified in `requirements.txt`, I used this manual hack. It's fixed now according to your suggestion.